app/src/main/java/jp/developer/retia/frozenword/ui/habbits/HabbitsScreen.kt

```diff
@@ -24,9 +24,30 @@ fun PreviewHabbits() {
         Surface {
             Habbits(
                 listOf(
-                    HabbitAndLog(Habbit(title = "hogehoge", trigger = "まるまるしたら"), emptyList()),
-                    HabbitAndLog(Habbit(title = "hogehoge", trigger = "まるまるしたら"), emptyList()),
-                    HabbitAndLog(Habbit(title = "hogehoge", trigger = "まるまるしたら"), emptyList()),
+                    HabbitAndLog(
+                        Habbit(
+                            title = "hogehoge",
+                            trigger = "まるまるしたら",
+                            simpleHabbitTitle = "何何する"
+                        ),
+                        emptyList()
+                    ),
+                    HabbitAndLog(
+                        Habbit(
+                            title = "hogehoge",
+                            trigger = "まるまるしたら",
+                            simpleHabbitTitle = "なになにする"
+                        ),
+                        emptyList()
+                    ),
+                    HabbitAndLog(
+                        Habbit(
+                            title = "hogehoge",
+                            trigger = "まるまるしたら",
+                            simpleHabbitTitle = "naninaniする"
```

Comment:
```suggestion
                        ),
                        emptyList()
```



<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>